### ISSUE TITLE
react-native windows command fails because there is no react-native-windows version compatible with the latest RN

### ISSUE THREAD

**Issue by rigdern** [ReactWindows/react-native-windows#778]
### Repro Steps

I followed the instructions for Getting Started in the README. After running `react-native windows`, I got this error:

```
Installing react-native-windows@0.35.*...
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save" "react-native-windows@0.35.*"
npm ERR! node v6.5.0
npm ERR! npm  v3.10.3

npm ERR! No compatible version found: react-native-windows@0.35.*
npm ERR! Valid install targets:
npm ERR! 1.0.0, 0.35.0-rc.0, 0.34.1, 0.34.0, 0.34.0-rc.0, 0.33.4, 0.33.3, 0.33.1, 0.33.0-rc.1, 0.33.0-rc.0, 0.32.1, 0.32.0, 0.32.0-rc.0, 0.31.3, 0.31.2, 0.31.1, 0.31.0, 0.31.0-rc.1, 0.31.0-rc.0, 0.30.0, 0.30.0-rc.0, 0.29.2, 0.29.1, 0.29.0, 0.29.0-rc.0, 0.28.3, 0.28.2, 0.28.1, 0.28.0, 0.28.0-rc.1, 0.28.0-rc.0, 0.27.0, 0.27.0-rc, 0.2.5-rc7, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.1.0
```

### Cause

I have `react-native` version `0.35.0` installed. I guess this is erroring because it's trying to install `react-native-windows` version `0.35.*` but this version pattern isn't compatible with any of the available `react-native-windows` versions based on the rules of semantic versioning.

### Work Around

Developers can work around this issue by running this command:

```
react-native windows --windowsVersion 0.35.0-rc.0
```

**Comment by rozele** [comment]
I still haven't published react-native-windows@0.35 yet.

On Oct 12, 2016, at 10:47 PM, Adam Comella <notifications@github.com<mailto:notifications@github.com>> wrote:


I followed the instructions for Getting Started in the README. After running react-native windows, I got this error:

Installing react-native-windows@0.35.*...
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save" "react-native-windows@0.35.*"
npm ERR! node v6.5.0
npm ERR! npm  v3.10.3

npm ERR! No compatible version found: react-native-windows@0.35.*
npm ERR! Valid install targets:
npm ERR! 1.0.0, 0.35.0-rc.0, 0.34.1, 0.34.0, 0.34.0-rc.0, 0.33.4, 0.33.3, 0.33.1, 0.33.0-rc.1, 0.33.0-rc.0, 0.32.1, 0.32.0, 0.32.0-rc.0, 0.31.3, 0.31.2, 0.31.1, 0.31.0, 0.31.0-rc.1, 0.31.0-rc.0, 0.30.0, 0.30.0-rc.0, 0.29.2, 0.29.1, 0.29.0, 0.29.0-rc.0, 0.28.3, 0.28.2, 0.28.1, 0.28.0, 0.28.0-rc.1, 0.28.0-rc.0, 0.27.0, 0.27.0-rc, 0.2.5-rc7, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.1.0


I have react-native version 0.35.0 installed. I guess this is erroring because it's trying to install react-native-windows version 0.35.* but this version pattern isn't compatible with any of the available react-native-windows versions based on the rules of semantic versioning.

**Comment by rigdern** [comment]
I figured that was the cause.

I want to point out that when RN UWP is behind RN in terms of published versions, RN UWP users will have a bad experience when creating new projects. I ran into this error message by following the [Getting Started](https://github.com/ReactWindows/react-native-windows#getting-started) instructions. Some of my concerns with the experience:
  - The error message isn't clear to new users. Maybe they'll retry running the steps and run into the same result. Maybe they'll give up.
  - Even if the user understands the problem, they may not know what to do to fix it.

Perhaps we can resolve these concerns by improving the error message the tool outputs. It can say that there is no version of `react-native-windows` that is compatible with the version of `react-native` they're using. It can list the newest version of `react-native` that `react-native-windows` is compatible with. For users trying to make a new project, it can suggest a command for them to run which will create a new RN project using the newest version of `react-native` that `react-native-windows` is compatible with. For users that are trying to add `react-native-windows` to an existing app, I guess we can suggest that they wait for a new version of `react-native-windows` to be published. I'm not sure what else we can do for them. At least they'll have the information about what the newest version of `react-native` is that is compatible with `react-native-windows`. They can try downgrading RN if they want.
